libs/core/src/lib/file-uploader/file-uploader.service.ts

```diff
@@ -9,6 +14,36 @@ export interface FileUploadOutput {
 
 @Injectable({ providedIn: 'root' })
 export class FileUploaderService {
+    static _parseFileSize(fileSize: string): number {
```

Comment:
Could you move this method to utils? I see it's used in upload collection, maybe it will be handy in other component as well